TrashMob/client-app/src/components/EventManagement/ShareToSocialsModal.tsx

```diff
@@ -1,168 +1,206 @@
-
-							</div>
-						}
-
-						<hr />
-
-						Share a link
-
-						<div className="row" id="iconsModalWrapper">
-							<div className="d-flex justify-content-between">
-								<div className="iconWrapper modalIcon" id="firstWrapper">
-									<FacebookShareButton
-										className={"socials-modal-icon"}
-										url={eventLink}
-										hashtag="#litter"
-									>
-										<FacebookIcon size={32} round />
-									</FacebookShareButton>
-								</div>
-								<div className="iconWrapper modalIcon">
-									<TwitterShareButton
-										className={"socials-modal-icon"}
-										title={parseShareMessage("twitter")}
-										hashtags={["litter"]}
-										url={eventLink}
-										via="TrashMobEco"
-									>
-										<TwitterIcon size={32} round />
-									</TwitterShareButton>
-								</div>
-								{/* missing: instagram integration */}
-								<div className="iconWrapper modalIcon">
-									<LinkedinShareButton
-										className={"socials-modal-icon"}
-										url={eventLink}
-									>
-										<LinkedinIcon size={32} round />
-									</LinkedinShareButton>
-								</div>
-								<div className="iconWrapper modalIcon">
-									<WhatsappShareButton
-										className={"socials-modal-icon"}
-										url={eventLink}
-										title={parseShareMessage("whatsapp")}
-									>
-										<WhatsappIcon size={32} round />
-									</WhatsappShareButton>
-								</div>
-								<div className="iconWrapper modalIcon">
-									<EmailShareButton
-										className={"socials-modal-icon"}
-										url={eventLink}
-										subject={props.emailSubject ?? `Join me at this TrashMob.eco event in ${props.eventToShare.city}!`}
-										body={parseShareMessage("email")}
-									>
-										<EmailIcon size={32} round />
-									</EmailShareButton>
-								</div>
-							</div>
-						</div>
-
-						<EventLink />
-					</Modal.Body>
-					<Modal.Footer>
-						<Button variant="secondary" onClick={handleClose}>
-							Close
-						</Button>
-					</Modal.Footer>
-				</Modal>
-			</div>
-		</>
-	);
-}
\ No newline at end of file
+  const [copiedLink, setCopied] = useState(false);
+  const eventLink =
+    props.eventLink ??
+    `${window.location.origin}/eventdetails/${props.eventToShare.id}`;
+  const eventDate = props.eventToShare
+    ? new Date(props.eventToShare.eventDate).toLocaleDateString("en-us", {
+        year: "numeric",
+        month: "2-digit",
+        day: "2-digit",
+      })
+    : "";
+  const eventTime = props.eventToShare
+    ? new Date(props.eventToShare.eventDate).toLocaleTimeString("en-us", {
+        hour12: true,
+        hour: "numeric",
+        minute: "2-digit",
+      })
+    : "";
+
+  const renderCopyToClipboardToolTip = (
+    <Tooltip id="tooltip">{ToolTips.CopyToClipboard}</Tooltip>
+  );
+
+  const handleClose = () => {
+    props.handleShow(false);
+    setCopied(false);
+  };
+
+  const handleCopyLink = () => {
+    if (copiedLink) {
```

Comment:
Would `if(!copiedLink) { ... }` make this more concise?

Reply:
I tried to stay in scope on this PR, I didn't touch non-UI related code unless absolutely necessary. The only reason this is showing a change is because Prettier reorganized it.

Reply:
Yep, that makes sense. 

Closing comment.